demos/benney_luke/benney_luke.py.rst

```diff
@@ -0,0 +1,198 @@
+Benney-Luke equations: a reduced water wave model

```

Comment:
The DOS line ending on this file confuse pylit which doesn't produce runnable output.  I can fix this up locally before merging.